### PR TITLE
Rework the README a bit, add PR template and feedback form

### PR DESCRIPTION
**Pull request by droberts-ada**

### CODE REVIEW

**Comment by droberts-ada** [comment]
I messed with indentation a bit so the README changes look bigger than they are.

File: README.md [hunk 1]
```diff
@@ -1,29 +1,42 @@
...
 
+## The API
 
-## Minimum Requirements 
-  - **Wave 1** Click button or link to show all trips
-  - **Wave 2** Can click on a trip, from the list of trips, to see more information about that trip
-    - Should be able to see id, name, destination, continent, about, category, weeks and cost 
-  - **Wave 3** While viewing a single trip, you can reserve a spot
-    - Use a form to submit your name to reserve a spot on the trip you are viewing
+View the [Travel API's documentation](https://github.com/ada-c6/trip_api).
```
Comment (Hamled) [inline README.md]: Should this link be updated to an Ada-C7 fork, or possibly just use the AdaGold repo?

File: README.md [hunk 2]
```diff
@@ -1,29 +1,42 @@
...
-Reminder: You shall use HTML best-practices. You should organize your HTML by grouping content using semantic HTML tags.
+- Reading data from an API using AJAX
+- Posting data to an API using AJAX
+- Building dynamic elements on a page with jQuery
+- Using Underscore templates to simplify rendering HTML from JavaScript
 
-## The API
+## Description
+
+You will build a web app that will query data from an api, using AJAX, to display a list or trips and view more information on a specific trip. While viewing a specific trip, you can claim your spot by sending data back to the API.
```
Comment (kariabancroft) [inline README.md]: "list of trips"

File: README.md [hunk 3]
```diff
@@ -1,29 +1,42 @@
...
+View the [Travel API's documentation](https://github.com/AdaGold/trip_api).
+
+## Requirements
+- **Wave 1** Click button or link to show all trips
+- **Wave 2** Can click on a trip, from the list of trips, to see more information about that trip
+  - Should be able to see id, name, destination, continent, about, category, weeks and cost
+- **Wave 3** While viewing a single trip, you can reserve a spot
+  - Use a form to submit your name to reserve a spot on the trip you are viewing
+- **For All Waves**
+  - Any dynamic elements on you page shall be rendered using Underscore templates
```
Comment (kariabancroft) [inline README.md]: your page? or the page?

File: README.md [hunk 4]
```diff
@@ -1,29 +1,42 @@
...
 
 
 ## Optional Enhancements
-  - **For more practice working with data**, filter trips by search queries (like by continent, budget, etc.). You'll need to explore API to see what functionality exists. 
-  - **To practice a more advanced POST**, allow the user to create a new trip. 
-  - **For more jQuery practice**, use jQuery to sort list of trips by specific attributes, like budget or time remaining. 
-  - **For an extra challenge**, have the number of spots available update on a page automatically if another user were to reserve a spot while you were viewing that trip.
-  
+
+- **For more practice working with data**, filter trips by search queries (like by continent, budget, etc.). You'll need to explore API to see what functionality exists.
```
Comment (kariabancroft) [inline README.md]: "Explore the API"

**Pull request merged**